src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react';
+import './css/App.css';
+import Header from './Header';
+import Timer from './Timer';
+import Image from './Image';
+import {TimerState, TimerType, TimerConstants, MAX_SESSION_TIME, MIN_SESSION_TIME, MAX_BREAK_TIME, MIN_BREAK_TIME} from './Constants';
+
+class App extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            breakLength:TimerConstants.breakLength,
+            sessionLength:TimerConstants.sessionLength,
+            type: TimerType.SESSION,
+            timerState: TimerState.NOTSTARTED
+        }
+        this.increaseSessionLength = this.increaseSessionLength.bind(this);
+        this.decreaseSessionLength = this.decreaseSessionLength.bind(this);
+        this.increaseBreakLength = this.increaseBreakLength.bind(this);
+        this.decreaseBreakLength = this.decreaseBreakLength.bind(this);
+    }
+
+    reset = () => {
+        this.setState({
+            breakLength:TimerConstants.breakLength,
+            sessionLength:TimerConstants.sessionLength,
+            type: TimerType.SESSION,
+            timerState: TimerState.RESET
+        });
+    }
+
+    increaseSessionLength = () => {
+        if(this.state.timerState !== TimerState.START) {
+            this.setState({sessionLength: (this.state.sessionLength < MAX_SESSION_TIME )?this.state.sessionLength+1:this.state.sessionLength});
```

Comment:
1. take this logic outside setState
```javascript
(this.state.sessionLength < MAX_SESSION_TIME )?
```
2. if the session length indeed has reached max time, you are unnecessarily setting state. I would suggest removing this logic and instead enable disable buttons so that it does not exceed max or min time